locales/en/data-classes.ftl

```diff
@@ -63,6 +63,7 @@ imsi-numbers = IMSI numbers
 income-levels = Income levels
 instant-messenger-identities = Instant messenger identities
 ip-addresses = IP addresses
+job-applications: Job applications
```

Comment:
This needs an equal sign ;-)

Reply:
Bah, sorry. I've been playing w/ YAML all morning and am asleep at the wheel.
Never trust a QA to do a Dev's work. Fixed.